src/screens/ProfileScreen/index.js

```diff
@@ -40,6 +40,7 @@ import useProfileScreenHook, {
 } from '../../hooks/screen/useProfileScreenHook';
 import {useAfterInteractions} from '../../hooks/useAfterInteractions';
 import {Analytics} from '../../libraries/analytics/firebaseAnalytics';
+import {deleteAnonymousPost, deletePost, viewTimePost} from '../../service/post';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `deleteAnonymousPost`, `deletePost`, and `viewTimePost` from `../../service/post` has been moved. Ensure that these functions are not used before their import, as this could lead to a ReferenceError.
```diff
- import {deleteAnonymousPost, deletePost, viewTimePost} from '../../service/post';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->